### PR TITLE
LLB Refactor

### PR DESCRIPTION
**Pull request by bensenescu**

### CODE REVIEW

File: src/router/router.js [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+import AboutUsView from '../views/AboutUsView.vue';
+import EventsView from '../views/EventsView.vue';
+import LoginView from '../views/LoginView.vue';
+import SignUpView from '../views/SignUpView.vue';
+import AccessControlDemonstration from '../components/AccessControl/AccessControlDemonstration.vue';
+import NotFoundView from '../views/NotFoundView.vue';
+import ProfileView from '../views/ProfileView.vue';
+import MyEventsView from '../views/MyEventsView.vue';
+import CheckoutView from '../views/CheckoutView.vue';
+import SingleEventView from '../views/SingleEventView.vue';
```
Comment (ZachOBrien) [inline src/router/router.js]: All those views looking lovely

File: src/views/CheckoutView.vue [hunk 2]
```diff
@@ -13,18 +13,23 @@
...
+import EventsListScroll from '../components/Events/EventsListScroll.vue';
 
 export default {
   name: 'Checkout',
   components: {
-    Events,
+    EventsListScroll,
+  },
+  computed: {
+    registeredEvents() {
```
Comment (ZachOBrien) [inline src/views/CheckoutView.vue]: How is this going to be used in the future? Will it be used to fetch Events from the store that this user is registered for?

File: src/components/AccessControl/AccessControl.vue [hunk 3]
```diff
@@ -16,7 +16,7 @@ export default {
   computed: {
     hasRole() {
       return (this.roles.length === 0 || this.roles.includes(this.role))
-          || (this.component_user_id === this.user_id);
+          || (this.componentUserId === this.userId);
```
Comment (ZachOBrien) [inline src/components/AccessControl/AccessControl.vue]: thanks for fixing this

File: src/views/CheckoutView.vue (hunk 2 above)
Comment (bensenescu) [reply src/views/CheckoutView.vue]: It will be held in the store because register events will be held in browser until they are purchased.

**Pull request merged**